org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NullTestCase.xtend

```diff
@@ -0,0 +1,60 @@
+package org.uqbar.project.wollok.tests.interpreter
+
+import org.junit.Test
+
+class NullTestCase extends AbstractWollokInterpreterTestCase {
+	
+	@Test
+	def void messageSentToNull() {
+		'''
+		assert.throwsException({ null.sayHi() })
+		assert.throwsException({ null.toString() })
+		'''.test
+	}
+
+	@Test
+	def void numberOperatorsSentToNull() {
+		'''
+		assert.throwsException({ null + 3 })
+		assert.throwsException({ null - 3 })
+		assert.throwsException({ null * 3 })
+		assert.throwsException({ null / 3 })
+		'''.test
+	}
+	
+	@Test
+	def void booleanOperatorsSentToNull() {
+		'''
+		assert.throwsException({ null || true })
+		assert.throwsException({ null && true })
+		'''.test
+	}
+	
+	@Test
+	def void equalEqualOperatorSentToNull() {
+		'''
+		var valorNulo
+		assert.notThat(null == 8)
```

Comment:
added lot of comparison in tests. It is tedious to define a single test, so excuse me for this:

``` java
	@Test
	def void equalEqualOperatorSentToNull() {
		'''
		class Golondrina { var energia = 0 }
		
		program a {
			var valorNulo
			assert.notThat(null == 8)
			assert.notThat(null == "pepe")
			assert.notThat(null == 3.0)
			assert.notThat(null == 1..2)
			assert.notThat(null == [1,2,3])
			assert.notThat(null == #{1,2,3})
			assert.notThat(null == assert)
			assert.notThat(null == new Golondrina())
			assert.notThat(valorNulo == 8)
			assert.that(valorNulo == null)
			assert.notThat(valorNulo != null)
		}
		'''.interpretPropagatingErrors
	}

	@Test
	def void equalEqualOperatorWithANullArgument() {
		'''
		class Golondrina { var energia = 0 }
		
		program a {
			assert.notThat(8 == null)
			assert.notThat("pepe" == null)
			assert.notThat(false == null)
			assert.notThat(2.0 == null)
			assert.notThat(1..2 == null)
			assert.notThat([1,2,3] == null)
			assert.notThat(#{1,2,3} == null)
			assert.notThat(assert == null)
			assert.notThat(new Golondrina() == null)
		}
		'''.interpretPropagatingErrors
	}
```

All tests are green